examples/petsc/bps.c

```diff
@@ -74,7 +74,7 @@ int main(int argc, char **argv) {
   if (ierr) return ierr;
   comm = PETSC_COMM_WORLD;
 
-  // Read CL options
+  // Read command line options
   ierr = PetscOptionsBegin(comm, NULL, "CEED BPs in PETSc", NULL); CHKERRQ(ierr);
```

Comment:
```suggestion
  ierr = PetscOptionsBegin(comm, NULL, "CEED Surface Area Example", NULL); CHKERRQ(ierr);
```

Reply:
This was actually a small doc change for consistency in the BPs file, so the docstring was ok, but thanks!